Client/mods/deathmatch/logic/luadefs/CLuaResourceDefs.cpp

```diff
@@ -18,7 +20,7 @@ void CLuaResourceDefs::LoadFunctions()
         {"call", Call},
         {"getThisResource", GetThisResource},
         {"getResourceConfig", GetResourceConfig},
-        {"getResourceName", GetResourceName},
+        {"getResourceName", ArgumentParser<GetResourceName>},
```

Comment:
This makes the function backward incompatible. Because there're many scripts that expect this function returns false in case of failure. You should you `ArgumentParserWarn<false, GetResourceName>` here.

Reply:
Thank you.